### PR TITLE
terrascan argo-cd instructions

### PR DESCRIPTION
**Pull request by storebot**

### CODE REVIEW

**Review by cesar-rodriguez — DISMISSED** [review]

File: docs/cicd.md [hunk 1]
```diff
@@ -52,3 +52,129 @@ terrascan:
...
+  kubectl create secret generic ssh-key-secret \
+    --from-file=ssh-privatekey=/path/to/.ssh/id_rsa \
+    --from-file=ssh-publickey=/path/to/.ssh/id_rsa.pub
+```
+
+Configuring the job to delete only after the specified time see `ttlSecondsAfterFinished` will allow users to check for violations in the UI w/o having to use the notification, see example below.
+
+![picture](img/terrascan-argo-cd-resource-hook-logs.png)
+
+Below if a full example on building the terrascan-argo-cd integration container.
```
Comment (jlk) [inline docs/cicd.md]: Below *is*

File: docs/cicd.md [hunk 2]
```diff
@@ -52,3 +52,129 @@ terrascan:
...
+  echo "notification exit code: $?"
+else
+  echo "no response skipping"
+fi
+```
+
+`Dockerfile`
+
+``` SH
+  FROM accurics/terrascan:929e377
```
Comment (jlk) [inline docs/cicd.md:156]: Can we use image tags, and not SHAs?

**Review by acc-jon — COMMENTED** [review]
Looks good to me

**Review by jlk — APPROVED** [review]
LGTM. Looks like the right tag's there but not shown in the PR somehow??

**Pull request merged**